### ISSUE TITLE
"Unable to find event" error after sending TX

### ISSUE THREAD

**Issue by raykyri** [hicommonwealth/commonwealth#676]
Replication steps:

- yarn reset-server
- Go to Edgeware (testnet), and link an account with balance on Beresheet
- Go to your profile page
- Set an on-chain identity name
- After setting the identity, you should get this error:

![image](https://user-images.githubusercontent.com/1273926/91506026-f3fd9c80-e885-11ea-90c9-befaf5d5c846.png)

The identity appears to be set correctly regardless of the error.

**Comment by jnaviask** [comment]
Was able to reproduce this both in browser and CLI. Talking with Jaco about it.

**Comment by jnaviask** [comment]
Additional info: first call of setIdentity succeeds but emits the following event:
```
{"ApplyExtrinsic":2} : utility.BatchInterrupted [0,{"Other":null}]
```

Later calls return the error as seen in the screenshot.

**Comment by jnaviask** [comment]
Fixed in commit 88dbc65117109ab0f16abb6f368375d0fb41f175 -- needed to update @edgeware/node-types.
